package-lock.json

```diff
@@ -115,7 +115,8 @@
         "ansi-escapes": {
             "version": "1.4.0",
             "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-1.4.0.tgz",
-            "integrity": "sha1-06ioOzGapneTZisT52HHkRQiMG4="
```

Comment:
We want to keep lock files out of packages per https://github.com/hapijs/contrib/issues/114 👍